R/codemeta_readme.R

```diff
@@ -54,20 +61,22 @@ ropensci_reviews <- memoise::memoise(.ropensci_reviews)
 guess_ropensci_review <- function(readme) {
 
   url <- "github.com/ropensci/onboarding/issues/"
+  url2 <- "github.com/ropensci/software-review/issues/"
 
-  badges <- get_badge_links_matching(readme, url)
+  badges <- get_badge_links_matching(readme, paste0(url, "|", url2))
 
   if (is.null(badges)) {
 
     return(NULL)
   }
 
-  review <- as.numeric(stringr::str_remove(badges, paste0(".*https://", url)))
+  url_m <- which_url_matches_badge_link(readme, c(url, url2))
+  review <- as.numeric(stringr::str_remove(badges, paste0(".*https://", url_m)))
 
   if (review %in% ropensci_reviews()$review) {
 
     list("@type" = "Review",
-         "url" = paste0("https://", url, review),
+         "url" = paste0("https://", url_m, review),
```

Comment:
shouldn't the URL to review always be "software-review", even when the badge was "onboarding"?

Reply:
oh right, good point

Reply:
thanks